examples/retrieve_results/bet_disk.py

```diff
@@ -1,61 +1,132 @@
+import json
 import os
 
-import flow360.component.v1.units as u
-import flow360.v1 as fl
+from pylab import show
+
+import flow360 as fl
 from flow360.examples import BETDisk
```

Comment:
rename BETExampleData
```suggestion
from flow360.examples import BETExampleData
```

Reply:
done